features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/instrumentation/gestures/WindowCallbackWrapper.kt

```diff
@@ -26,7 +26,7 @@ internal class WindowCallbackWrapper(
     window: Window,
     val sdkCore: SdkCore,
     val wrappedCallback: Window.Callback,
-    val gesturesDetector: GesturesDetectorWrapper,
+    private val gesturesDetector: GesturesDetectorWrapper,
     val interactionPredicate: InteractionPredicate = NoOpInteractionPredicate(),
     val copyEvent: (MotionEvent) -> MotionEvent = {
```

Comment:
Can we add the same fixes in the `WindowCallbackWrapper` for SR please ? Maybe add it in this PR

Reply:
ah no...sorry didn't go through all the PR 😢 . My bad, tks a tons for the update will review the whole PR later.